config/functions

```diff
@@ -386,6 +386,8 @@ endian = '$endian'
 root = '$root'
 $(python -c "import os; print('c_args = {}'.format([x for x in os.getenv('CFLAGS').split()]))")
 $(python -c "import os; print('c_link_args = {}'.format([x for x in os.getenv('LDFLAGS').split()]))")
+$(python -c "import os; print('cpp_args = {}'.format([x for x in os.getenv('CXXFLAGS').split()]))")
```

Comment:
mixing cpp and cxx looks wrong.
should it more like 'cpp_args -> CPPFLAGS' and 'cxx_args -> CXXFLAGS' ?

Reply:
oh holy...
you are right: meson uses cpp_args for c++. really confusing